views/apply-success.njk

```diff
@@ -0,0 +1,26 @@
+{% extends "./layouts/main.njk" %}
+
+{% block pageTitle %}
+  Successful Application
+{% endblock %}
+
+{% block links %}
+<link rel="stylesheet" href="/css/style.css">
+{% endblock %}
+
+{% block content %}
+
+
+<div class="d-flex justify-content-center">
+  <div>
```

Comment:
Unnecessary div

Reply:
Same as other comment but removing this div uncenters all of the content on the page